### ISSUE TITLE
Fontsize

### ISSUE THREAD

**Issue by Aksword** [radimitrov/CSharpShellApp#213]
Hello, the System.drawing is available on the app, but when I want to call the Font function and change the font and its size, I get an error message. Does someone have a clue about how to change the font size ?
Thanks !

**Comment by jgranserver** [comment]
i think its TextSize if im not wrong

**Comment by Aksword** [comment]
I searched and text-size is for CSS, all I found for C# is the Font class (which doesn't work), and the Labell. , which is only available in the Windows.Forms library.

**Comment by radimitrov** [comment]
Font size? Are we talking about the console? And what error message? PlatformNotImplementedException?
System.Drawing.Common is what is in the app, just data structures. Android doesn't have the features Windows does in System.Drawing.
I'll add a FontSize property to the app's Console class.

**Comment by Aksword** [comment]
To change the font size, all I found on the net was to create a new Font. So I wrote : Font test = New Font("Arial", 24);
And I got :
System.PlatformNotSupportedException: Operation is not supported on this platform.
  at System.Drawing.Font..ctor (System.String familyName, System.Single emSize) [0x00006] in <e870ece2ce9144d8b736cce3406c9f55>:0 
  at (wrapper remoting-invoke-with-check) System.Drawing.Font..ctor(string,single)
  at Emojisgame.Program.Main () [0x00000] in <f73d81bc0a4a4e7eb7c8eebc0b77c37f>:0 
  at (wrapper managed-to-native) System.Reflection.RuntimeMethodInfo.InternalInvoke(System.Reflection.RuntimeMethodInfo,object,object[],System.Exception&)
  at System.Reflection.RuntimeMethodInfo.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x0006a] in <02432465956f42cebcacad2804482ab1>:0 
Maybe you know another way to do that, and thanks for your next adding !

**Comment by radimitrov** [comment]
Yes, pretty much what I expected. Not implemented for Android. Probably only the basics are.
Additionally, it seems Microsoft are removing that package entirely for non-Windows platforms.
https://learn.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/6.0/system-drawing-common-windows-only

**Comment by radimitrov** [comment]
If we're talking about platform specific equavelents then a Font is a [Typeface ](https://developer.android.com/reference/android/graphics/Typeface.html)in Android.
And the [Android.Graphics](https://learn.microsoft.com/en-us/dotnet/api/android.graphics?view=xamarin-android-sdk-12) namespace contans stuff like Bitmap and Canvas.
These are all in Mono.Android.dll
